script.js

```diff
@@ -1,4 +1,310 @@
+// What is the input going to be? Stone or Paper or Scissors
+// What should the output be? You played {input} and the computer played {stone/paper/scissors} so you {won/lost/drew}
+// How will the computer's option be randomly generated? Using math.random to generate a number between 0-2
+// How many different cases are there? 3 (win, lose, draw)
+
+var userWins = 0;
+var compWins = 0;
+var draws = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // Generate a random dice number
+  var randomDiceNumber = rollDice();
+  // Assign a play to each dice number
+  if (randomDiceNumber == 0) {
+    var computerTurn = "stone";
+    console.log("computer played 0 - stone");
+    console.log(randomDiceNumber == 0);
+  }
+
+  if (randomDiceNumber == 1) {
+    console.log("computer played 1 - paper");
+    console.log(randomDiceNumber == 1);
+    var computerTurn = "paper";
+  }
+
+  if (randomDiceNumber == 2) {
+    console.log("computer played 2 - scissors");
+    console.log(randomDiceNumber == 2);
+    var computerTurn = "scissors";
+  }
```

Comment:
Would suggest putting this in a separate function, e.g. var generateComputerPlay, outside of the main function, and then calling that function in the main function.